Description: Add a print to System.err when some bundle is missing (will improve diagnostic when some symlink is broken).
 This patch is not strictly necessary but it makes it easier to find packaging errors.
Author: Damien Raude-Morvan <drazzib@debian.org>
Forwarded: not-needed
--- a/biz.aQute.bndlib/src/aQute/bnd/build/Project.java
+++ b/biz.aQute.bndlib/src/aQute/bnd/build/Project.java
@@ -1295,8 +1295,11 @@
 						File file = plugin.get(bsn, version, attrs, blocker);
 						// and the entry must exist
 						// if it does, return this as a result
-						if (file != null)
+						if (file != null) {
 							return toContainer(bsn, range, attrs, file, blocker);
+						} else {
+							System.err.println("Missing: " + bsn + " " + range);
+						}
 					}
 				}
 			}
